### ISSUE TITLE
Failure with rlang 0.2.2

### ISSUE THREAD

**Issue by lionel-** [numeract/Nmisc#8]
Hello, I'm about to release rlang 0.2.2 and I see your package fails the revdep checks.

You can get this version with

```r
remotes::install_github("r-lib/rlang", ref = "release-0.2.2")
```

I see:

```
✖ | 16 1     | Testing project_package.R [0.3 s]
────────────────────────────────────────────────────────────────────────────
test_project_packages.R:169: failure: generate_install_file works
`nchar_expected` not equal to `nchar_install_package`.
1/1 mismatches
[1] 84 - 76 == 8
```

I don't understand what's going on but it doesn't seem like rlang is at fault.

**Comment by MikeBadescu** [comment]
@lionel- Thank you for reporting this and for your work on `rlang`! This particular test expects `rlang` (and the other packages) to be installed from CRAN, not from anywhere else. Once `rlang` 0.2.2 will be available on CRAN, this test will pass - thus, there is no need to postpone the `rlang` release. 

I agree, it is a strange edge case and I will update the test in the near future to avoid this and similar issues.

**Comment by MikeBadescu** [comment]
issue fixed in 0.3.3, closing
